ci-operator/step-registry/configure-registry-storage/destroy-s3-cloudfront/configure-registry-storage-destroy-s3-cloudfront-commands.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# This script will destroy s3 bucket and cloudfront that step configure-registry-storage-deploy-s3-cloudfront generated.
+
```

Comment:
@xiuwang  since `create_s3_bucket_for_registry_storage.tf` is moved to `/tmp`, so I suggest adding `trap 'rm -f /tmp/create_s3_bucket_for_registry_storage.tf' EXIT TERM INT` here